cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -614,8 +614,8 @@ var __AdyenHelper: Object = {
      * Checks if payment method is blocked
      */
     isMethodTypeBlocked : function(methodType) {
-        if (methodType.indexOf('bcmc') !== -1 ||
-            (methodType.indexOf('wechatpay') !== -1 && methodType.indexOf('wechatpayWeb') === -1) ||
+        if (methodType == "bcmc" || methodType == "bcmc_mobile_QR" ||
+            (methodType.indexOf("wechatpay") !== -1 && methodType.indexOf("wechatpayWeb") === -1) ||
```

Comment:
no because there are some other wechatpay payment methods that we want to include in the unsupported payment methods list. In prestashop we listed all the method names and didn't just check the first part of the payment methods. Personally I like this approach more: https://github.com/Adyen/adyen-prestashop/blob/1.2.0/adyen.php#L1036 Easier to understand and gives a better overview of the payment methods.